docker/ssl/Makefile

```diff
@@ -11,29 +11,29 @@ all: clean_certs ca server_crt
 # CA name and key is "ca".
 ca:
 	openssl req -newkey rsa:2048 -x509 -nodes -sha512 -days 1095 \
-				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=localhost/O=Mainflux/OU=IoT/emailAddress=info@mainflux.com"
+				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=$(CN)/O=$(O)/OU=$(OU)/emailAddress=$(EA)"
```

Comment:
Docs that describe the use of this script should reflect this change
so we should put something like
`make ca CN=localhost O=Mainflux OU=IoT`

Reply:
Did a PR.